data-node/src/main/java/org/graylog/datanode/filesystem/index/indexreader/ShardStatsParserImpl.java

```diff
@@ -16,16 +16,14 @@
  */
 package org.graylog.datanode.filesystem.index.indexreader;
 
+import jakarta.inject.Singleton;
+import org.apache.lucene.index.IndexFormatTooOldException;
+import org.apache.lucene.index.StandardDirectoryReader;
+import org.apache.lucene.store.Directory;
+import org.apache.lucene.store.FSDirectory;
+import org.apache.lucene.util.Version;
 import org.graylog.datanode.filesystem.index.IncompatibleIndexVersionException;
 import org.graylog.datanode.filesystem.index.IndexerInformationParserException;
-import org.graylog.shaded.opensearch2.org.apache.lucene.index.DirectoryReader;
-import org.graylog.shaded.opensearch2.org.apache.lucene.index.IndexFormatTooOldException;
-import org.graylog.shaded.opensearch2.org.apache.lucene.index.StandardDirectoryReader;
-import org.graylog.shaded.opensearch2.org.apache.lucene.store.Directory;
-import org.graylog.shaded.opensearch2.org.apache.lucene.store.FSDirectory;
-import org.graylog.shaded.opensearch2.org.apache.lucene.util.Version;
```

Comment:
Can you add a forbiddenapis rule to ban the `org.graylog.shaded.opensearch2.org.apache.lucene` imports? I guess using those in our code is always an error?

Reply:
thanks. yes, I will add a check